app/models/client.rb

```diff
@@ -1,45 +1,57 @@
 class Client
-  # include Elasticsearch::Model
   include Elasticsearch::Model::Callbacks
+  include Elasticsearch::Model::Proxy
   include Elasticsearch::Persistence::Model
   include ActiveModel::Validations
 
   include Cacheable
   include Indexable
-
-
-  attribute :symbol,  String,  mapping: { type: 'keyword', analyzer: "keyword", tokenizer: "keyword" }
```

Comment:
Clients symbol have `dots` in the name and thus need to be analysed as *Keywords* this goes from the moment of indexing to search. I don't remember if I validated that works without stating the analyser.